include/deal.II/meshworker/dof_info.h

```diff
@@ -272,15 +278,25 @@ namespace MeshWorker
 //----------------------------------------------------------------------//
 
   template <int dim, int spacedim, typename number>
-  DoFInfo<dim,spacedim,number>::DoFInfo(const DoFHandler<dim,spacedim> &dof_handler)
+  DoFInfo<dim, spacedim, number>::DoFInfo(const dealii::DoFHandler<dim, spacedim> &dof_handler)
     :
```

Comment:
Why do you add the `dealii::` namespace here? We should be inside of it and it counteracts the idea behind `DEAL_II_NAMESPACE_OPEN`. I am not sure though we still support compilers that need the macro